src/containers/offerContainer/offerContainer.jsx

```diff
@@ -0,0 +1,21 @@
+import Box from '@mui/material/Box'
+import OfferCard from '~/components/offer-card/OfferCard'
+
+
+const OfferContainer = ({ offerCards, viewMode }) => {
+
```

Comment:
Git says you 'viewMode' is defined but never used

Reply:
fixed